src/convert.cpp

```diff
@@ -4,6 +4,20 @@
 #include <array>
 #include <cmath>
 #include <vector>
+#include <string>
+#include <sstream>
+
+namespace {
+
+template <typename T> std::string to_string(const T& value) {
+    std::ostringstream stream;
+    stream.setf(std::ios::fixed);
+    stream.precision(6);
+    stream << value;
+    return stream.str();
+}
```

Comment:
No, let's keep it like that.

Ideally we'd use rapidjson's [number stringification](https://github.com/miloyip/rapidjson/blob/master/include/rapidjson/internal/dtoa.h) though to avoid having the slowness of `stringstream`.